bump-native-dd-sdk.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
```

Comment:
I added a bash script that will allow us to automatically update all needed files when bumping a native sdk. All we need is pass the platform and the new version to it and it will update all version numbers on the needed files.